e2e-tests/insurance/cypress/e2e/journeys/export-contract/agent/save-and-back.spec.js

```diff
@@ -0,0 +1,116 @@
+import FIELD_IDS from '../../../../../../constants/field-ids/insurance/export-contract';
+import { INSURANCE_ROUTES } from '../../../../../../constants/routes/insurance';
+
+const {
+  ROOT,
+  ALL_SECTIONS,
+  EXPORT_CONTRACT: { PRIVATE_MARKET },
+} = INSURANCE_ROUTES;
+
+const {
+  PRIVATE_MARKET: { ATTEMPTED: FIELD_ID },
+} = FIELD_IDS;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Export contract - Agent - Save and go back', () => {
+  let referenceNumber;
+  let url;
+  let allSectionsUrl;
+
+  before(() => {
+    cy.completeSignInAndGoToApplication({ totalContractValueOverThreshold: true }).then(({ referenceNumber: refNumber }) => {
+      referenceNumber = refNumber;
+
+      // go to the page we want to test.
+      cy.startInsuranceExportContractSection({});
+      cy.completeAndSubmitAboutGoodsOrServicesForm({});
+      cy.completeAndSubmitHowYouWillGetPaidForm({});
+
+      url = `${baseUrl}${ROOT}/${referenceNumber}${PRIVATE_MARKET}`;
+      allSectionsUrl = `${baseUrl}${ROOT}/${referenceNumber}${ALL_SECTIONS}`;
+
+      cy.assertUrl(url);
+    });
+  });
+
+  beforeEach(() => {
+    cy.saveSession();
+  });
+
+  after(() => {
+    cy.deleteApplication(referenceNumber);
+  });
+
+  describe('when submitting an empty form via `save and go back` button', () => {
+    beforeEach(() => {
+      cy.navigateToUrl(url);
+
+      cy.clickSaveAndBackButton();
+    });
+
+    it(`should redirect to ${ALL_SECTIONS}`, () => {
+      cy.assertUrl(allSectionsUrl);
+    });
+  });
+
+  describe(`when selecting no for ${FIELD_ID}`, () => {
+    it(`should redirect to ${ALL_SECTIONS}`, () => {
+      cy.navigateToUrl(url);
+
+      cy.completeAndSubmitAgentForm({ usingAgent: false });
```

Comment:
should this not be `cy.completeAgentForm`?

Reply:
Whoops, great spot